firefox_puppeteer/api/prefs.py

```diff
@@ -202,3 +202,13 @@ def set_pref(self, pref_name, value):
             """, script_args=[pref_name, value])
 
         assert retval
+
+    def persist_prefs(self):
+        """Persist current pref values to the default prefs file (typically prefs.js)
+        """
+        with self.marionette.using_context('chrome'):
+            self.marionette.execute_script("""
+              Cu = Components.utils;
+              Cu.import("resource://gre/modules/Services.jsm");
+              Services.prefs.savePrefFile(null);
```

Comment:
That's what actually is happening here. As reference here the bug which I filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1137388